Mathlib/SetTheory/PGame/Order.lean

```diff
@@ -156,9 +156,9 @@ instance : Preorder PGame :=
           (x ≤ y → y ≤ z → x ≤ z) ∧ (y ≤ z → z ≤ x → y ≤ x) ∧ (z ≤ x → x ≤ y → z ≤ y) from
         fun x y z => this.1
       intro x y z
-      induction' x with xl xr xL xR IHxl IHxr generalizing y z
-      induction' y with yl yr yL yR IHyl IHyr generalizing z
-      induction' z with zl zr zL zR IHzl IHzr
+      induction x generalizing y z with | mk xl xr xL xR IHxl IHxr => ?_
+      induction y generalizing z with | mk yl yr yL yR IHyl IHyr => ?_
+      induction z with | mk zl zr zL zR IHzl IHzr => ?_
```

Comment:
The loss of symmetry here is rather unfortunate.

Reply:
Maybe @vihdzp cares (or doesn't)

Reply:
Let us just ignore this purely aesthetic loss.

Reply:
@eric-wieser Violeta did not respond, what now?